src/builder-git.c

```diff
@@ -578,6 +578,16 @@ builder_git_mirror_repo (const char     *repo_location,
                     origin, full_ref_mapping, NULL))
             return FALSE;
 
+          g_print ("Fetching LFS assets\n");
+          if (!git (mirror_dir, NULL, 0, error,
+                    "lfs", "fetch", "--all", NULL))
+            {
+              git (mirror_dir, NULL, 0, error,
+                   "lfs", "logs", "last", NULL);
```

Comment:
This potentially passes a set error as argument to the second `git` call. Same below.